hushline/__init__.py

```diff
@@ -126,3 +146,43 @@ def add_onion_location_header(response: Response) -> Response:
                     app.logger.warning("HostOrganization data not found in database.")
 
     return app
+
+
+def register_commands(app: Flask) -> None:
+    stripe_cli = AppGroup("stripe")
+
+    @stripe_cli.command("configure")
+    def configure() -> None:
+        """Configure Stripe and premium tiers"""
+        # Make sure tiers exist
+        with app.app_context():
+            free_tier = db.session.query(Tier).filter_by(name="Free").first()
+            if not free_tier:
```

Comment:
Bigger philosophical question that came in the last PR. Do we want migrations to handle this sort of thing or app code? I feel like pre-populating tables should go in a migration so we're guaranteed that if the schema exists it has correct values.

Reply:
I will choose to ignore philosophy in the service of merging this PR sooner lol